tests/ui/specs/MediaPlayer/MediaPlayer-specs.js

```diff
@@ -6,6 +6,12 @@ describe('MediaPlayer', function () {
 		Page.open();
 	});
 
+	function waitForPlayMedia (mediaPlayer, timeout) {
```

Comment:
It is recommend that you move this function to MediaPlayerPage.js to make it easier for other tests to use.